### DISCUSSION TITLE
Use Cordova plugins in Vue File not works

### DISCUSSION THREAD

**Discussion by carlosaroca** [quasarframework/quasar#11723] [CLI - Capacitor / Cordova modes]
When I try to use Cordova in file allways show error **"Uncaught (in promise) ReferenceError: cordova is not defined"** Im use like this with Quasar 2..3 but this not works.

I try other ways but not works any. Please some support for this, in Quasar 1 this works normal

I just use like 
```
after export default defineComponent({
setup () {
    const $q = cordova.plugins
    console.log("HOLA")
    console.log($q)
    console.log("CHAO")
  } 
.
.
.

```

**Comment by MilosPaunovic** [comment]
Take a look at the official docs example of the Cordova plugins and guides on how to use them:
[https://quasar.dev/quasar-cli/developing-cordova-apps/cordova-plugins#introduction](https://quasar.dev/quasar-cli/developing-cordova-apps/cordova-plugins#introduction)

**Reply by realtebo** [reply]
there isn't any refrr to `cordova.` syntax in the examples.

**Comment by realtebo** [comment]
1. did you find a solution?
2. can you explain me what does it mean this syntax?
```
after export default defineComponent({
```
